src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -700,7 +632,7 @@ <CD> void invalidateAllChannelsInBlob(
 
   /** Get the server generated unique prefix for this client */
   String getClientPrefix() {
```

Comment:
does flushService still need this method which is just passing the call onto storageManager? Or can callers directly call storageManager.getClientPrefix instead?

Reply:
Removed and call directly from a `StorageManager`